lib/client.js

```diff
@@ -161,7 +163,7 @@ class CloudantClient {
     request.abort = false;
     request.clientCallback = callback;
 
-    request.clientStream = new PassThroughDuplex();
+    request.clientStream = new PassThroughDuplex(ReplayableStream, stream.PassThrough);
```

Comment:
It might be nice to do this only if the thing is actually being piped and there are plugins configured to avoid the overheads of the `ReplayableStream` in the simplest use cases. I think that optimisation can be safely added later though.